build-tools/create-packs/Microsoft.Android.Sdk.proj

```diff
@@ -38,37 +38,41 @@ sdk pack imported by Microsoft.NET.Workload.Android.
+      <AndroidSdkBuildTools Include="@(_MSBuildFilesWin)" Condition=" '$(AndroidHostRID)' == 'windows-x64' ">
+        <RelativePath>$([MSBuild]::MakeRelative($(MSBuildSrcDir), %(FullPath)))</RelativePath>
+      </AndroidSdkBuildTools>
       <!-- Exclude host-os specific native libraries, aside from libMonoPosixHelper.dylib which is required by LibZipSharp on macOS. -->
-      <AndroidSdkBuildToolsUnix Include="@(_MSBuildFilesUnix);@(_MSBuildFilesUnixSign);@(_MSBuildFilesUnixSignAndHarden)" >
+      <AndroidSdkBuildTools Include="@(_MSBuildFilesUnix);@(_MSBuildFilesUnixSign);@(_MSBuildFilesUnixSignAndHarden)" Condition=" '$(AndroidHostRID)' == 'linux-x64' or '$(AndroidHostRID)' == 'osx-x64' ">
         <RelativePath>$([MSBuild]::MakeRelative($(MSBuildSrcDir), %(FullPath)))</RelativePath>
-      </AndroidSdkBuildToolsUnix>
-      <AndroidSdkBuildToolsUnix Include="$(PkgMono_Posix_NETStandard)\runtimes\osx\native\libMonoPosixHelper.dylib" Condition="$([MSBuild]::IsOSPlatform('osx'))">
+      </AndroidSdkBuildTools>
+      <AndroidSdkBuildTools Include="$(PkgMono_Posix_NETStandard)\runtimes\osx\native\libMonoPosixHelper.dylib" Condition=" '$(AndroidHostRID)' == 'osx-x64' ">
         <RelativePath>libMonoPosixHelper.dylib</RelativePath>
-      </AndroidSdkBuildToolsUnix>
-      <AndroidSdkBuildToolsUnix Include="$(PkgMono_Posix_NETStandard)\runtimes\linux-x64\native\libMonoPosixHelper.so" Condition="$([MSBuild]::IsOSPlatform('linux'))" >
+      </AndroidSdkBuildTools>
+      <AndroidSdkBuildTools Include="$(PkgMono_Posix_NETStandard)\runtimes\linux-x64\native\libMonoPosixHelper.so" Condition=" '$(AndroidHostRID)' == 'linux-x64' " >
         <RelativePath>libMonoPosixHelper.so</RelativePath>
-      </AndroidSdkBuildToolsUnix>
+      </AndroidSdkBuildTools>
+      <!-- Remove items with '%(ExcludeFromAndroidNETSdk)' == 'true' metadata -->
+      <AndroidSdkBuildTools Remove="@(AndroidSdkBuildTools)" Condition=" '%(AndroidSdkBuildTools.ExcludeFromAndroidNETSdk)' == 'true' " />
     </ItemGroup>
-    <!-- Copy only the required tools to new folder. Skip Unix item copying on Windows. -->
+
+    <RemoveDir Directories="$(ToolsSourceDir)" />
     <Copy
         SourceFiles="@(AndroidSdkBuildTools)"
         DestinationFiles="@(AndroidSdkBuildTools->'$(ToolsSourceDir)%(RelativePath)')"
-        SkipUnchangedFiles="True"
-    />
-    <Copy
-        SourceFiles="@(AndroidSdkBuildToolsUnix)"
-        DestinationFiles="@(AndroidSdkBuildToolsUnix->'$(ToolsSourceDir)%(RelativePath)')"
-        SkipUnchangedFiles="True"
-        Condition="$([MSBuild]::IsOSPlatform('osx')) or $([MSBuild]::IsOSPlatform('linux'))"
     />
+
     <ItemGroup>
+      <!-- Microsoft.Android.Sdk.ILLink output -->
+      <_PackageFiles Include="$(XAInstallPrefix)xbuild\Xamarin\Android\Microsoft.Android.Sdk.ILLink.dll" PackagePath="tools" />
+      <_PackageFiles Include="$(XAInstallPrefix)xbuild\Xamarin\Android\Microsoft.Android.Sdk.ILLink.pdb" PackagePath="tools" />
```

Comment:
@radekdoulik I've updated the output directory for `Microsoft.Android.Sdk.ILLink.csproj` so that we don't include extra files that are automatically copied to the output dir that aren't required.

Are there any other files that I am missing that we need to include in the SDK pack? I did some diffing locally and these are the only other assemblies that seemed like they _could_ be related, but the .NET 6 tests we have that use the linker all seem to be working with only `Microsoft.Android.Sdk.ILLink.dll`:

 * Mono.Cecil.dll
 * Mono.Cecil.Mdb.dll
 * Mono.Cecil.Pdb.dll
 * Mono.Cecil.Rocks.dll
 * System.Text.Json.dll

Reply:
If the tests are working, then it should be fine. The Cecil and Json probably come with the ILLink and the `Java.Interop.Export` and `Xamarin.Android.Build.Tasks` are already included?

Reply:
Ok sounds good, Java.Interop.Export and Xamarin.Android.Build.Tasks are added via their explicit inclusions in `create-installers.targets`: https://github.com/xamarin/xamarin-android/blob/master/build-tools/installers/create-installers.targets#L142
https://github.com/xamarin/xamarin-android/blob/master/build-tools/installers/create-installers.targets#L241